### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by julienben**
I noticed it's been 3 months since the 1.2.0 release so I thought let's do a bit of maintenance and release 1.3.0. What do you think?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,20 +61,17 @@
...
     "hooks": {
       "pre-commit": "lint-staged"
     }
   },
   "lint-staged": {
-    "*.js": [
-      "eslint --fix",
-      "git add"
-    ]
+    "*.js": "eslint --fix"
```
Comment (BenLorantfy) [inline package.json]: not `git add` ? I find it's better DX.

Comment (julienben) [reply package.json]: `lint-staged` does it automatically now.

https://github.com/okonet/lint-staged/releases/tag/v10.0.0

Comment (BenLorantfy) [reply package.json]: oh cool 👍

**Comment by julienben** [comment]
@BenLorantfy How do I release a new version of the code to npm? Do I need to run `npm run publish` locally? And after that I can create the release here?

**Comment by BenLorantfy** [comment]
Yep. And then just put `v1.3.0` in the tag version field.

**Comment by BenLorantfy** [comment]
We might want to setup a way to automatically publish versions (maybe via GitHub actions) without having to do so locally

**Comment by BenLorantfy** [comment]
Oh wait: we should merge dev to master first

**Comment by julienben** [comment]
All done!

**Pull request merged**